bitnami/mysql/templates/secrets.yaml

```diff
@@ -13,9 +13,9 @@ metadata:
   {{- end }}
 type: Opaque
 data:
-  mysql-root-password: {{ include "mysql.root.password" . | b64enc | quote }}
-  mysql-password: {{ include "mysql.password" . | b64enc | quote }}
+  mysql-root-password: {{ include "common.secrets.passwords.manage" (dict "secret" (include "common.names.fullname" .) "key" "mysql-root-password" "length" 10 "providedValues" (list "auth.rootPassword") "context" $) }}
```

Comment:
Great! With this change we can read secrets during upgrades and we don't need to set the value `root.password`. Could your review NOTES.txt file and change validations/messages about this topic? An example (Sorry I can't comment in GH this part):
```console
To upgrade this helm chart:

  1. Obtain the password as described on the 'Administrator credentials' section and set the 'root.password' parameter as shown below:

      ROOT_PASSWORD=$(kubectl get secret --namespace default mysql -o jsonpath="{.data.mysql-root-password}" | base64 --decode)
      helm upgrade --namespace default mysql bitnami/mysql --set auth.rootPassword=$ROOT_PASSWORD
```